js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/PolicyEvaluationTable.tsx

```diff
@@ -318,15 +328,20 @@ const ViewDetailsButton = ({
           setShowDetails(false);
         }}
       >
-        <AssetEventMetadataEntriesTable showDescriptions event={evaluation} repoAddress={null} />
+        <Box padding={8}>
+          <AssetEventMetadataEntriesTable showDescriptions event={evaluation} repoAddress={null} />
+        </Box>
+        <DialogFooter>
+          <Button onClick={() => setShowDetails(false)}>Done</Button>
+        </DialogFooter>
       </Dialog>
-      <Button
+      <ButtonLink
         onClick={() => {
           setShowDetails(true);
         }}
       >
         View details
-      </Button>
+      </ButtonLink>
```

Comment:
The `Button` doesn't fit well within the evaluation rows, and the table spacing is incorrect for the dialog body. Just trying to tidy it up a bit here.